test/k8sT/Services.go

```diff
@@ -1035,6 +1035,31 @@ var _ = Describe("K8sServicesTest", func() {
 			testExternalTrafficPolicyLocal()
 		})
 
+		SkipContextIf(
+			func() bool {
+				return helpers.RunsWithoutKubeProxy()
+			},
```

Comment:
Nit: this could be shortened as:

```suggestion
			helpers.RunsWithoutKubeProxy,
```

Reply:
Ditto